samples/net/secure_mqtt_sensor_actuator/src/tls_config/cert.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CERT_H__
+#define __CERT_H__
+
+/* The CA certficate of the MQTT broker should be included here
+ *  The certificate can either be in DER or PEM format.
+ *  A DER certificate can be converted to a byte array using
+ *  "cat ca.crt | sed -e '1d;$d' | base64 -d |xxd -i"
+ *  If using a PEM certifificate, each line should be wrapped in "\r\n"
+ */
+
+/* CA certificate for test.mosquitto.org
+ * (valid at time of development, may require updating)
```

Comment:
Would be nice to include source of this certificate.

Reply:
I added the source - test.mosquitto.org